package.json

```diff
@@ -8,7 +8,7 @@
   ],
   "scripts": {
     "client": "yarn workspace client start",
-    "server": "yarn workspace server start",
+    "server": "NODE_ENV=development yarn workspace server start",
```

Comment:
I don't think we need this anymore are we?

Reply:
Gooood point :+1: